packages/pwa-buildpack/src/WebpackTools/plugins/__tests__/UpwardPlugin.spec.js

```diff
@@ -227,7 +227,7 @@ test('dev-mode IOAdapter uses fetch', async () => {
     );
 });
 
-test('dev-mode IOAdapter can fetch unsecure URLs', async () => {
+test('dev-mode IOAdapter can fetch insecure URLs', async () => {
```

Comment:
Unsecure refers more to the ability to have security, but simply choosing not to have it. Insecure means that you're without security, in any situation.

Unsecure is the proper, less ambiguous term to use here.